vendor/bat-native-ads/src/bat/ads/internal/base/strings/string_conversions_util.cc

```diff
@@ -18,4 +22,27 @@ std::string BoolToString(const bool value) {
   return value ? kTrue : kFalse;
 }
 
+std::vector<float> ConvertStringToVector(std::string string) {
+  const std::vector<std::string> vector_string = base::SplitString(
+      string, " ", base::TRIM_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
+  std::vector<float> vector;
+  for (const std::string& element_string : vector_string) {
+    double element;
+    base::StringToDouble(element_string, &element);
+    vector.push_back(static_cast<float>(element));
+  }
+
+  return vector;
+}
+
+std::string ConvertVectorToString(std::vector<float> vector) {
+  size_t v_index = 0;
+  std::vector<std::string> vector_as_string;
```

Comment:
This is a vector of strings, and only when we return is it a vector_as_string. Can we consider a different name? Thanks